broadway/broadway-bundle/0.3/config/packages/broadway.yaml

```diff
@@ -0,0 +1 @@
+broadway: ~
```

Comment:
We don't add "empty" config files to avoid cluttering the config directory.

Reply:
are those rules somewhere written down? Did I miss this?

Reply:
I'm going to add an auto-validation for this if possible and a note in the README.